factories/users.rb

```diff
@@ -46,5 +46,9 @@
     factory :translation_admin do
       roles { [Role.find_or_create_by(name: "translation_admin")] }
     end
+
+    factory :official do
```

Comment:
Can we call this `official_user`, just in case we end up calling other things `official` in the future?